build.sbt

```diff
@@ -808,9 +751,10 @@ lazy val zincScripted = (project in internalPath / "zinc-scripted")
 lazy val crossTestBridges = {
   Command.command("crossTestBridges") { state =>
     val publishCommands = List(
-      s"${compilerBridgeTest210.id}/test",
-      s"${compilerBridgeTest211.id}/test",
+      // s"${compilerBridgeTest210.id}/test",
```

Comment:
Why don't we test 210 and 211 in `crossTestBridges`?

Reply:
Earlier they were failing, but I got them both to pass now.